### PR TITLE
Set scrollbar properties for Firefox 64+

### PR DESCRIPTION
**Pull request by Scrxtchy**
copies https://github.com/thelounge/thelounge/commit/a9eced888fae9c561030e606e1021fac0f534f0d#diff-53a2103ea4ad1eb199a9906db330e16bR14

image

Refs:
https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-color
https://drafts.csswg.org/css-scrollbars-1/

### CODE REVIEW

**Comment by MiniDigger** [comment]
oh, after 18 years they finally did it? :joy: 

not sure if thats the color we want to go for.
how about `rgba(255, 255, 255, 0.06)`, its the color of the unread badge:
image

File: mininapse.css [hunk 1]
```diff
@@ -393,3 +393,8 @@ code {
 #upload-progressbar {
     background: var(--highlight-border-color);
 }
+
+/* Scrollbars */
+body {
+    scrollbar-color: var(--highlight-border-color) var(--body-bg-color);
```
Comment (MiniDigger) [inline mininapse.css]:
```suggestion
    scrollbar-color: rgba(255, 255, 255, 0.06) var(--body-bg-color);
```

Comment (Scrxtchy) [reply mininapse.css]: It should have at least a 3:1 contrast ratio to comply with WCAG

Comment (MiniDigger) [reply mininapse.css]: mmmh. I am not a pro on accessibility but doesn't that guideline only apply to text? 

as a user with no visual impairment that red is way too distracting for me, not sure how others (@fnutt ?) feel. I personally don't use firefox so I don't mind whatever color we choose, chrome already has very dim scroll bars.

Comment (Scrxtchy) [reply mininapse.css]: There's a sidenote on the CSS Scrollbars Module document

> When using scrollbar-color property with specific color values, authors should ensure the specified colors have enough contrast between them. For keyword values, UAs should ensure the colors they use have enough contrast. See Techniques for WCAG 2.0: G183: Using a contrast ratio of 3:1 […] [WCAG20].
https://www.w3.org/TR/2018/WD-css-scrollbars-1-20180925/#scrollbar-color

Comment (fnutt) [reply mininapse.css:399]: Sorry for late response.

I don't like the red colour that much either. It's very "in your face" for something that I would not like to be in my face.

**Comment by fnutt** [comment]
Closing due to negative comments from both @MiniDigger and myself. Thanks for the PR though.

**Pull request closed without merging**